src/axom/quest/detail/MarchingCubesImpl.hpp

```diff
@@ -266,22 +271,43 @@ class MarchingCubesImpl : public MarchingCubesSingleDomain::ImplBase
 
     axom::IndexType* crossingId = axom::allocate<axom::IndexType>(
       1,
-      axom::execution_space<ExecSpace>::allocatorID());
+      axom::detail::getAllocatorID<MemorySpace::Dynamic>());
     *crossingId = 0;
-    axom::for_all<ExecSpace>(
-      0,
-      parentCellCount,
-      AXOM_LAMBDA(axom::IndexType n) {
-        auto caseId = caseIdsView.flatIndex(n);
-        auto ccc = num_contour_cells(caseId);
-        if(ccc != 0)
+
+    auto loopBody = AXOM_LAMBDA(axom::IndexType n)
```

Comment:
Loop body is declared ahead of use, so it doesn't have to be repeated for the RAJA case and the non-RAJA case.